recipes/openquake/meta.yaml

```diff
@@ -0,0 +1,290 @@
+{% set version = "2.8.0" %}
+
+package:
+  name: openquake.engine
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/o/openquake.engine/openquake.engine-{{ version }}.tar.gz
+  sha256: 55aed57bb65395e733106e5a1b2cb3fde9d8966f40f3b02167ccc9446d356473
+
+build:
+  number: 0
+  skip: True  # [not py35]
```

Comment:
The reason it was "building" all Pythons was b/c you did not added the skip, but b/c you make it more restrictive below nothing was really built, the CIs were only initializing.